lib/alavetelitheme.rb

```diff
@@ -1,3 +1,4 @@
+# -*- encoding : utf-8 -*-
```

Comment:
This misses a bunch of files:

- https://github.com/mysociety/dirittodisapere-theme/blob/6ed760cfc358c14a04f8bb499b3e50f6c7a1d56f/uninstall.rb
- https://github.com/mysociety/dirittodisapere-theme/blob/6ed760cfc358c14a04f8bb499b3e50f6c7a1d56f/post_install.rb